solr/contrib/ltr/src/java/org/apache/solr/ltr/rest/ManagedModelStore.java

```diff
@@ -190,20 +155,22 @@ public LTRScoringAlgorithm makeLTRScoringAlgorithm(Map<String,Object> map)
     final List<Feature> features = new ArrayList<>();
     final List<Normalizer> norms = new ArrayList<>();
     for (final Object modelFeature : featureList) {
-      try {
-        // check the declared features exist in the feature store
-        final Map<String,Object> modelFeatureMap = 
-            (Map<String,Object>) modelFeature;
-        final Feature feature = parseFeature(modelFeatureMap,
-            fstore);
-        final Normalizer norm = parseNormalizer(modelFeatureMap);
-        norms.add(norm);
-        features.add(feature);
-      } catch (FeatureException e) {
-        throw new SolrException(ErrorCode.BAD_REQUEST, e);
-      } catch (final CloneNotSupportedException e) {
-        throw new SolrException(ErrorCode.BAD_REQUEST, e);
+      final Map<String,Object> modelFeatureMap =
+          (Map<String,Object>) modelFeature;
+
+      final String featureName = (String) modelFeatureMap.get(CommonLTRParams.FEATURE_NAME);
+      final Feature feature = (featureName == null ? null : fstore.get(featureName));
```

Comment:
Discussed, agreed that null checking can happen here.  Potentially any feature name validation can happen in the validate() function that we plan on adding after the invokeSetters is called.